src/rpcclient.cpp

```diff
@@ -11,8 +11,10 @@
 #include <set>
 #include <stdint.h>
 
+#include <boost/algorithm/string/case_conv.hpp> // for to_lower()
```

Comment:
This is still present in upstream master, also with no apparent usage in the file.